### ISSUE TITLE
website: Add Aloglia based search to our docs.

### ISSUE THREAD

**Issue by bwplotka** [thanos-io/thanos#2629]
cc @thisisobate :hugs:

**Comment by thisisobate** [comment]
Nice! That would be a nice-to-have on our docs. Should I proceed with implementation? Or should we implement once we unblock the versioning issue?

**Comment by bwplotka** [comment]
Up to you really, this is just a side project to versioning really (:

**Comment by thisisobate** [comment]
Alright. Thanks for the clarification @bwplotka

**Comment by bwplotka** [comment]
This issue was fixed by here: https://github.com/thanos-io/thanos/pull/2664 thanks to @thisisobate :muscle:
